src/bun.js/api/server.zig

```diff
@@ -5297,6 +5304,44 @@ pub fn NewServer(comptime NamespaceType: type, comptime ssl_enabled_: bool, comp
             return JSC.JSValue.jsNumber(@as(i32, @intCast(@as(u31, @truncate(this.activeSocketsCount())))));
         }
 
+        pub fn getAddress(this: *ThisServer, globalThis: *JSGlobalObject, _: *JSC.CallFrame) callconv(.C) JSC.JSValue {
+            switch (this.config.address) {
+                .unix => |unix| {
+                    var value = bun.String.create(bun.sliceTo(@constCast(unix), 0));
+                    defer value.deref();
+                    return value.toJS(globalThis);
+                },
+                .tcp => {
+                    if (this.cached_localAddr.address.isEmpty()) {
```

Comment:
This should reuse the `JSC::Structure` object which I think @paperdave had done in earlier code for `requestIp` 

As-is, this will be pretty slow to create these objects if it's called on each request

Reply:
oh I'm misunderstanding, this is per server

it's probably fine, though a getter + cache: true is still a good idea